components/token_metrics/actions/get-ai-reports/get-ai-reports.mjs

```diff
@@ -0,0 +1,75 @@
+import tokenMetrics from "../../token_metrics.app.mjs";
+import { ENDPOINTS, FILTER_DEFINITIONS } from "../../common/constants.mjs";
+import { buildParams, generateFilterSummary } from "../../common/utils.mjs";
+
+const endpoint = ENDPOINTS.AI_REPORTS;
+
+export default {
+  key: "token_metrics-get-ai-reports",
+  name: "Get AI Reports",
+  description: `${endpoint.description}. [See the documentation](https://developers.tokenmetrics.com/reference/ai-reports)`,
+  version: "0.0.1",
+  type: "action",
+  props: {
+    tokenMetrics,
+    // Filter props based on endpoint configuration and API documentation
+    tokenId: {
+      ...FILTER_DEFINITIONS.token_id,
+      description: "Comma Separated Token IDs. Click here to access the list of token IDs. Example: 37493,3484",
```

Comment:
Example values should usually be encased in backticks for Markdown formatting, e.g.
> Example: `37493,3484`